### DISCUSSION TITLE
Support for Indian recognised and unrecognised languages

### DISCUSSION THREAD

**Discussion by ghost** [squidfunk/mkdocs-material#5438] [Ideas]
As a citizen of India and user of this tool with who use to share the content to wide end user base in India and to make it useful I need the Indian language support on Material docs which are missing. We can start with below listed languages and eventually cover all of the languages.

- Sanskrit
- Kannada
- Telugu
- Tamil
- Malayalam

Do you think this can worked on on general release as new supported language? Happy to work on this if there is approval by members and maintainers.

Note: If any one reading this may have any other Indian language suggestion please comment below.

**Comment by squidfunk** [comment]
Yes, you can propose separate PRs for those languages. PRs adding new languages are always welcome! We're currently working on a guide that better explains how to do this, but in the meantime, it's should be quite self-explanatory [if you can read Jinja code](https://github.com/squidfunk/mkdocs-material/blob/master/src/partials/languages/hi.html).

**Reply by ghost** [reply]
Cool, I'll take this up.
